storage/Storage.ts

```diff
@@ -159,6 +158,7 @@ export default class Storage {
             upgrade.createStore("photo");
             upgrade.createStore("nextOfKin");
             upgrade.createStore("carer");
+            upgrade.createStore("otherSupport");
```

Comment:
You need to bump the database version and do this migration in the new version, or it won't be reflected on devices that have already opened this database. Grab me tomorrow if you want to talk it through.